generated/testproto/grpc/dummy_pb2_grpc.pyi.expected

```diff
@@ -0,0 +1,64 @@
+"""
+@generated by mypy-protobuf.  Do not edit manually!
+isort:skip_file
+"""
+from abc import (
+    ABCMeta as abc___ABCMeta,
+    abstractmethod as abc___abstractmethod,
+)
+
+from grpc import (
+    Channel as grpc___Channel,
+    ServicerContext as grpc___ServicerContext,
+)
+
+from typing import (
+    Generator as typing___Generator,
+)
+
+
+from .dummy_pb2 import *
```

Comment:
eg here - if self._import_message is working correctly, then it should bring in the dependencies as needed.
Clearly something more tricky is going on. Ok if this isn't solved immediately, but adding a comment to the codegen to improve this would be nice.

Reply:
`self._import_message` is already used for all messages. As you see it does nothing for messages withing same proto file.